src/Microsoft.Tye.Extensions.Configuration/TyeSecretsConfigurationBuilderExtensions.cs

```diff
@@ -17,34 +17,26 @@ public static class TyeSecretsConfigurationBuilderExtensions
         /// Adds Tye's secrets to <see cref="IConfiguration" />.
         /// </summary>
         /// <param name="builder">The <see cref="IConfigurationBuilder" />.</param>
-        /// <param name="configure">A delegate for additional configuration.</param>
         /// <returns>The <see cref="IConfigurationBuilder" />.</returns>
         /// <remarks>
         /// The environment variable <c>TYE_SECRETS_PATH</c> is used to populate the directory used by secrets.
         /// When the environment variable is specified, and the specified directory exists, then the value of
         /// <see cref="TyeSecretsConfigurationSource.FileProvider" /> will be non-null.
         /// </remarks>
-        public static IConfigurationBuilder AddTyeSecrets(this IConfigurationBuilder builder, Action<TyeSecretsConfigurationSource>? configure = null)
+        public static IConfigurationBuilder AddTyeSecrets(this IConfigurationBuilder builder)
         {
-            TyeSecretsConfigurationSource source;
             var secretsDirectory = Environment.GetEnvironmentVariable(TyeSecretsConfigurationSource.TyeSecretsPathEnvironmentVariable);
-            if (secretsDirectory == null || !Directory.Exists(secretsDirectory))
+            if (Directory.Exists(secretsDirectory))

```

Comment:
?

Reply:
This was straight up broken. The config provider was totally busto, because the KPF-file provider doesn't search recursively.

What's here now matches the copy-paste code that we deleted a week agao.